images/eventlistenersink/make.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The pipelines-service Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR="$(
+  cd "$(dirname "$0")" >/dev/null
+  pwd
+)"
+
+WORK_DIR="$SCRIPT_DIR/work"
+
+init() {
+  rm -rf "$WORK_DIR"
+  mkdir -p "$WORK_DIR"
+}
+
+checkout_upstream_repo() {
+  git clone "https://github.com/tektoncd/triggers.git" "$WORK_DIR/upstream"
```

Comment:
it may be nice to avoid doing a clone each time. In another script I offer the option to set the location in an environment variable and only clone if it is not set